src/content/ai-agent-fleets/ai-agent-fleets-admin-config.mdx

```diff
@@ -0,0 +1,238 @@
+
+### Step 4: Configure Your LLM Provider
+
+AI Agent Fleets requires an LLM provider to power the Okteto AI Agents. Choose between Anthropic (direct) or via Amazon Bedrock.
+
+#### Option A: Anthropic Configuration
+Use this option for direct integration with Anthropic's API.
+
+<p align="center">
+  <Image
+    src={require("@site/static/img/admin_ai_llm_provider_config_anthropic.png").default}
+    alt="AI Agent Fleets LLM Provider Configuration"
+    width="800"
+  />
+</p>
+
+1. **Select Provider**: Choose **Anthropic** from the Provider dropdown
+2. Obtain API Key:
+  - Click the "Get your Anthropic API key →" link
+  - Sign in to your [Anthropic account](https://console.anthropic.com/) (or create one)
+  - Navigate to [API Keys section](https://console.anthropic.com/settings/keys)
+  - Create a new API key with appropriate permissions
+  - Copy the key
+
+3. **Enter API Key**: Click Update and paste your Anthropic API key in the field
+4. **Save Configuration**: Click Save to save your settings
+
+#### Option B: Amazon Bedrock Configuration
+Use this option if your organization prefers AWS-managed AI services.
+
+<p align="center">
+  <Image
+    src={require("@site/static/img/admin_ai_llm_provider_config_bedrock.png").default}
+    alt="AI Agent Fleets LLM Provider Configuration for Bedrock"
+    width="800"
+  />
+</p>
+
+1. **Select Provider**: Choose **Amazon Bedrock** from the Provider dropdown
+2. **Configure Region**:
+  - Select the AWS region where you have Bedrock enabled
+  - Common regions: `us-east-1`, `us-west-2`, `eu-west-1`
+  - Ensure Claude models are available in your selected region
+
+3. **Set Up AWS Credentials**:
+  - Click "Get your Bedrock API key →" for setup instructions
+  - You'll need either:
+    - AWS Access Key ID and Secret Access Key
+    - IAM role with appropriate Bedrock permissions
+
+4. **Required AWS Permissions**:
+```
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "bedrock:InvokeModel",
+        "bedrock:InvokeModelWithResponseStream"
+      ],
+      "Resource": "arn:aws:bedrock:*:*:model/anthropic.claude-*"
+    }
+  ]
+}
+```
+
+5. **Enter Credentials**: Add your AWS credentials in the API Key field
+6. **Save Configuration**: Click Save to complete the configuration
+
+### Step 5: Configure LLM Model
+The LLM Model section shows which Claude model is being used:
+- Currently, agents use Claude Sonnet 4, optimized for software development tasks
+- This model provides the best balance of speed and capability for code generation
+- Additional models will be available in future releases
+
+:::note
+Note: The model selection is currently managed by Okteto to ensure optimal performance. Custom model selection will be added in a future update.
+:::
+
+### Step 6: Set Up Git Integration
+Configure Git integration to enable agents to create pull requests and manage code:
```

Comment:
The most important think about the git token is the ability to clone private git repositories